setup/kubernetes/aws.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Set Up an Amazon Elastic Kubernetes Cluster
+description: Learn how to set up an Amazon EKS cluster for your Coder deployment.
+---
+
+This deployment guide shows you how to set up an Amazon Elastic Kubernetes
+Engine cluster on which Coder can deploy.
+
+## Prerequisites
+
+Before proceeding, please make sure that you have the following utilities
+installed on your machine:
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [AWS Command Line
+  Interface](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+  (you'll also need to
+  [configure](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html)
+  the the command line interface to interact with your AWS account; consider
+  AWS' [CLI configuration
+  quickstart](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html)
+  to fast-track this process))
```

Comment:
```suggestion
  to fast-track this process
```

I may be missing something but I think these are extra `))` that aren't needed? @khorne3 let me know :)